pkg/services/login/logintest/logintest.go

```diff
@@ -44,6 +45,10 @@ func (a *AuthInfoServiceFake) GetAuthInfo(ctx context.Context, query *models.Get
 	return a.ExpectedError
 }
 
+func (a *AuthInfoServiceFake) GetUserLabels(ctx context.Context, query models.GetUserLabelsQuery) (map[int64]string, error) {
+	return map[int64]string{int64(1): login.GetAuthProviderLabel(login.LDAPAuthModule)}, nil
```

Comment:
I think we should be able to configure the values returned here in the fake. See how the other methods is doing it.

Reply:
Changed as requested.